bin/create-ebs-volume

```diff
@@ -147,11 +147,12 @@ alphabet=( {a..z} )
 
 function get_next_logical_device() {
     for letter in ${alphabet[@]}; do
-        # use /dev/xvdb* device names to avoid contention for /dev/sd* and /dev/xvda names
-        # only supported by HVM instances
-        if [ ! -b "/dev/xvdb${letter}" ]; then
-            echo "/dev/xvdb${letter}"
-            break
+        if [[ ! -z $(cat /var/run/ebs_autoscale_device_used | grep -w $letter) ]]; then
```

Comment:
it's better not to use `[[` unless absolutely necessary. also it's better to always wrap things in quotes:

`if [ ! -z "$(cat ebs_autoscale_device_used | grep -w a)" ]; then ...`